### PR TITLE
Countries widget

### PR DESCRIPTION
**Pull request by mihaelabuda**

### CODE REVIEW

File: js/homepage.js [hunk 1]
```diff
@@ -4,14 +4,23 @@ var towns = ["Amsterdam", "London", "Paris", "Copenhagen", "Madrid", "Rome", "St
 var randomTown = Math.floor(Math.random() * 10);
 var randomInfo = towns[randomTown];
 
-window.onload = function widgetWeather() {
+window.onload = function widget() {
 	fetch('http://api.openweathermap.org/data/2.5/weather?q=' + randomInfo + '&appid=35c6b5008a4aee8c23d418344ed3e046')
```
Comment (scozma) [inline js/homepage.js]: @mihaelabuda - Please move the two 'fetch' blocks into their own special functions and call them inside the 'widget' function (maybe rename this to 'populateWidgets' or 'generateWidgets', or something similar)

File: js/homepage.js [hunk 2]
```diff
@@ -4,14 +4,23 @@ var towns = ["Amsterdam", "London", "Paris", "Copenhagen", "Madrid", "Rome", "St
...
 	.then(function(randomWeather){
```
Comment (scozma) [inline js/homepage.js]: @mihaelabuda - Please add 'catch' handlers to the 'fetch' response object. When you get an error, you should display a message to the bottom right corner about this, using the existing notification/error component.

File: js/homepage.js (hunk 1 above)
Comment (mihaelabuda) [reply js/homepage.js]: Hi @scozma - Changes applied.

File: js/homepage.js (hunk 2 above)
Comment (mihaelabuda) [reply js/homepage.js]: Hi @scozma - Changes applied.

**Pull request merged**